controllers/machinedeletionremediation_controller.go

```diff
@@ -304,10 +316,7 @@ func (r *MachineDeletionRemediationReconciler) getMachine(remediation *v1alpha1.
 		}
 	}
 
-	machine := new(unstructured.Unstructured)
-	machine.SetKind(machineKind)
-	machine.SetAPIVersion(v1beta1.SchemeGroupVersion.String())
-
+	machine := new(v1beta1.Machine)
```

Comment:
nit: another one from the not-used-to-it category: at all other places where we need to have a pointer to a new empty object for the client.Get() call, we use machine := &v1beta1.Machine{}. I don't know why, I don't see a difference. But maybe just for consistency use the common pattern? :thinking:

Reply:
I picked this from some example somewhere, but yes I agree on consistency